src/components/app/Home.js

```diff
@@ -50,7 +51,9 @@ class InstallButton extends React.PureComponent<InstallButtonProps> {
 }
 
 type UploadButtonProps = {
-  retrieveProfileFromFile: typeof retrieveProfileFromFile,
+  retrieveProfileFromFile: WrapFunctionInDispatch<
+    typeof retrieveProfileFromFile
+  >,
```

Comment:
On `OwnProps` and props that consume an already wrapped ThunkAction creator, expect to see this `WrapFunctionInDispatch<Fn>` type.